server/boot/user.js

```diff
@@ -450,6 +465,33 @@ module.exports = function(app) {
     });
   }
 
+  function showResetProgress(req, res) {
+    return res.render('account/reset-progress', { title: 'Reset My Progress!'
+    });
+  }
+
+  function postResetProgress(req, res, next) {
+    User.findById(req.user.id, function(err, user) {
+      if (err) { return next(err); }
+      return user.updateAttributes({
+        progressTimestamps: [{
+          timestamp: Date.now()
+        }],
+        currentStreak: 0,
+        longestStreak: 0,
+        currentChallengeId: '',
+        isBackEndCert: false,
+        isFullStackCert: false,
```

Comment:
Is there a 'isFrontEndCert' and/or 'isDataVizCert' that would need reset as well? Or something like that?